### PR TITLE
Bump version for motion demos

### PR DESCRIPTION
**Pull request by clocksmith**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: gallery
 description: A resource to help developers evaluate and use Flutter.
 repository: https://github.com/flutter/gallery
-version: 2.4.0+020400
+version: 2.4.1
```
Comment (guidezpl) [inline pubspec.yaml:4]: Needs to be 2.4.1+020401

Comment (rami-a) [reply pubspec.yaml:4]: yeah, this format was needed to make the play store happy

Comment (rami-a) [reply pubspec.yaml:4]: Actually since this includes all those nice new motion demos, maybe 2.5.0 is more appropriate?

File: pubspec.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name: gallery
 description: A resource to help developers evaluate and use Flutter.
 repository: https://github.com/flutter/gallery
-version: 2.4.0+020400
+version: 2.5.0+020401
```
Comment (rami-a) [inline pubspec.yaml:4]:
```suggestion
version: 2.5.0+020500
```

Comment (clocksmith) [reply pubspec.yaml]: Done. How are these values after the + determined?

**Pull request merged**